internal/store/store.go

```diff
@@ -146,10 +146,6 @@ func sendItems(target Target, keyCh <-chan Key, doneCh <-chan struct{}, logger l
 
 // StreamItems reads the keys from the store and replays the corresponding item to the target.
 func StreamItems[I any](store Store[I], target Target, doneCh <-chan struct{}, logger logger) {
```

Comment:
we have to gofunc the callers in all the other places too. We have bucket notification targets using store.StreamItems()